Szymon/src/test/java/zadanie6/page/RegisterPage.java

```diff
@@ -0,0 +1,191 @@
+package zadanie6.page;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class RegisterPage extends LeftNavigationMenu {
+
+    private String password;
```

Comment:
Te pola to tez nie jest element strony - powinienes stworzyc inna klase User, ktora reprezentowalaby uzytkownika z tymi wszystkimi polami, getterami i setterami. Klasa Page to reprezentacja GUIa, a nie danych testowych.

```
public class User {
    private String username;
    private String firstName;
    private String lastName;
    private String addres;
    private String city;
    private String state;
    private String zipCode;
    private String PhoneNumber;
    private String ssn;

public String getAddres() {
        return addres;
    }
     public void setAddres(String addres) {
        this.addres = addres;
    }
     public String getCity() {
        return city;
    }
     public void setCity(String city) {
        this.city = city;
    }
     public String getState() {
        return state;
    }
     public void setState(String state) {
        this.state = state;
    }
     public String getZipCode() {
        return zipCode;
    }
     public void setZipCode(String zipCode) {
        this.zipCode = zipCode;
    }
     public String getSsn() {
        return ssn;
    }
     public void setSsn(String ssn) {
        this.ssn = ssn;
    }
     public String getFirstName() {  //wiem, że nie tak
        return firstName;
    }
     private void setFirstName(String firstName) {
        this.firstName = firstName;
    }
     public String getLastName() {
        return lastName;
    }
     public String getPassword() {
        return password;
    }
     public void setPassword(String password) {
        this.password = password;
    }
     public String getPhoneNumber() {
        return PhoneNumber;
    }
     public void setPhoneNumber(String phoneNumber) {
        PhoneNumber = phoneNumber;
    }
     public String getUsername() {
        return username;
    }
     public void setUsername() {
        username = getRandomUsername(5);
    }
}
```